src/core/api/payment_manager_api/admin-panel/_base_policy.xml.tpl

```diff
@@ -11,7 +11,16 @@
           <method>HEAD</method>
         </allowed-methods>
       </cors>
-      <set-backend-service base-url="{{pm-gtw-hostname}}/pp-admin-panel" />
+      <set-variable name="fromDnsHost" value="@(context.Request.OriginalUrl.Host)" />
+      <choose>
+        <when condition="@(context.Variables.GetValueOrDefault<string>("fromDnsHost").Contains("prf.platform.pagopa.it"))">
+          <set-variable name="backend-base-url" value="@($"{{pm-host}}/pp-admin-panel")" />
+        </when>
+        <otherwise>
+          <set-variable name="backend-base-url" value="@($"{{pm-host-prf}}/pp-admin-panel")" />
+        </otherwise>
+      </choose>
```

Comment:
```suggestion
      <choose>
        <when condition="@(context.Variables.GetValueOrDefault<string>("fromDnsHost").Contains("prf.platform.pagopa.it"))">
          <set-variable name="backend-base-url" value="@($"{{pm-host-prf}}/pp-admin-panel")" />
        </when>
        <otherwise>
          <set-variable name="backend-base-url" value="@($"{{pm-host}}/pp-admin-panel")" />
        </otherwise>
      </choose>
```
???